### ISSUE TITLE
Change App Version in Registry to Match Github Tag

### ISSUE THREAD

**Issue by Trenly** [fontforge/fontforge#5178]
* Provide a general summary of the issue in the **Title** above.  
* Before you open an issue, please check if a similar issue already exists or has been closed before.

### When you open an issue for a change/improvement/feature request:
- [x] A description of the problem you're trying to solve, including _why_ you think this is a problem
  - The current versioning scheme used for Windows is simply the date the package was built. However, this causes issues when comparing version numbers. Specifically, it causes issues with the [winget cli](/microsoft/winget-cli) and sorting the version numbers. This is because, when sorting, 08-03-2022 is greater than 01-01-2023. Using this format for the date causes a standard version comparison to fail to the highest day, then the highest month, then the highest year. 
  
![image](https://user-images.githubusercontent.com/12611259/211902912-daef40a5-bc00-4b5d-8133-103d285798eb.png)

- [x] If the feature changes current behavior, reasons why your solution is better
  - I propose the DisplayVersion (key written to the registry) be updated to be `Year.Month.Day` so that versions can be sorted properly in winget, powershell, and other automation tools. 

- [ ] \(optional) Possible solution/fix/workaround

**Comment by JoesCat** [comment]
Reading https://github.com/microsoft/winget-cli/issues/2840, it appears to have self-resolved - this could probably be closed.

**Comment by Trenly** [comment]
> Reading [microsoft/winget-cli#2840](https://github.com/microsoft/winget-cli/issues/2840), it appears to have self-resolved - this could probably be closed.

I disagree. Simply because there are workarounds does not mean that the current way the version is writtent to the registry is proper

**Comment by ctrlcctrlv** [comment]
Using YMD in general is good practice even if M$ doesn't require it.

**Comment by jtanx** [comment]
I'll fix it for the next release.

**Comment by iorsh** [comment]
Fixed in https://github.com/jtanx/fontforgebuilds/commit/a5042a95bfe7a79f249b85b87b14485e6c75a69a
